lib/verk/supervisor.ex

```diff
@@ -16,10 +16,10 @@ defmodule Verk.Supervisor do
 
   @doc false
   def init(_) do
-    queues = Application.get_env(:verk, :queues, [])
+    queues = Confex.get(:verk, :queues, [])
```

Comment:
Ya we need to use Confex.get_map in this case and you dont actually dynamically define your  whole queues which kinda make sense, only the number of jobs being processed at a time:

```
config :verk, queues: [default: {:system, :integer, "VERK_QUEUE_DEFAULT_JOBS", 25}, priority: {:system, :integer, "VERK_QUEUE_PRIORITY_JOBS", 10}]
```

Then :
```
env VERK_QUEUE_DEFAULT_JOBS=50 iex -S mix
iex(1)> Confex.get_map(:verk, :queues)
[default: 50, priority: 10]
```